pyrene/src/components/TextArea/TextArea.tsx

```diff
@@ -112,7 +112,7 @@ const TextArea: React.FC<TextAreaProps> = ({
         className={clsx(styles.textArea, { [styles.resizeable]: resizeable }, { [styles.filled]: value })}
         name={name}
         placeholder={placeholder}
-        rows={rows}
+        rows={rows < 1 ? 1 : rows}
```

Comment:
I would suggest using `rows={Math.min(1, rows)}`

Reply:
> Negative numbers (and 0) should either not be valid, or behave like 1 imo.

The idea as stated by @dparret is to equate to 1 row if the number of `rows` passed is less than or equal to 0

> Math.min(1, -5)

This will return -5 instead of 1 as expected

Reply:
✅